setup.cfg

```diff
@@ -152,3 +152,23 @@ skip=
     asv_bench/benchmarks/dtypes.py
     asv_bench/benchmarks/strings.py
     asv_bench/benchmarks/period.py
+    pandas/__init__.py
```

Comment:
what i mean is, can you just run isort on these?

Reply:
o i guess could be a followup

Reply:
running isort on all the __init__.py files gives..

```
ImportError while loading conftest 'C:\Users\simon\OneDrive\code\pandas-simonjayhawkins\pandas\conftest.py'.
pandas\__init__.py:29: in <module>
    import pandas.arrays
pandas\arrays\__init__.py:6: in <module>
    from pandas.core.arrays import (
pandas\core\arrays\__init__.py:4: in <module>
    from .categorical import Categorical  # noqa
pandas\core\arrays\categorical.py:33: in <module>
    from pandas.core.base import NoNewAttributesMixin, PandasObject, _shared_docs
pandas\core\base.py:27: in <module>
    import pandas.core.nanops as nanops
pandas\core\nanops.py:54: in <module>
    set_use_bottleneck(get_option('compute.use_bottleneck'))
pandas\core\config.py:227: in __call__
    return self.__func__(*args, **kwds)
pandas\core\config.py:97: in _get_option
    key = _get_single_key(pat, silent)
pandas\core\config.py:83: in _get_single_key
    raise OptionError('No such keys(s): {pat!r}'.format(pat=pat))
E   pandas.core.config.OptionError: "No such keys(s): 'compute.use_bottleneck'"
```

so some work required to do this.

Reply:
gotcha ok then

Reply:
also i'm on Windows and not able to run isort 4.3.8 because of https://github.com/timothycrosley/isort/issues/835, so struggling to make progress on this.

Reply:
ok, can you post an issue about this

Reply:
do we need a separate issue? #23334 is still open. add to that?

Reply:
let's add to that one.